README.md

```diff
@@ -296,3 +296,29 @@ module.exports = class MyApiListData extends ApiListData {
 	}
 };
 ```
+
+### get customParameters()
+
+This allows you to set custom **query parameters** for your API, it must contain the names (keys) and and data type (values) of the custom parameters that the API will receive.
+
+For example:
+
+```js
+'use strict';
+
+const { ApiListData } = require('@janiscommerce/api-list');
+
+module.exports = class MyApi extends ApiListData {
+
+    get customParameters() {
+        return {
+			additionalParameter: 'string?' // This parameter can be optional
+		};
+	}
+}
+
+/* 
+    This will allow the API to use custom query parameters, example:
+    https://domain.com/api/my-api-list?additionalParameter='someData'
+*/
+```
```

Comment:
Acá nos falto como es que se accede a los parámetros desde la API, nos faltaría documentar que los parámetros llegan dentro del `this.data` y agregar un ejemplo de uso también 🥺